gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/PermissionService.groovy

```diff
@@ -53,4 +60,26 @@ class PermissionService {
       fiatService.sync()
     }
   }
+
+  List<String> getServiceAccounts(@SpinnakerUser User user) {
+    List<String> accounts = []
+    if (!user) {
+      log.debug("getServiceAccounts: Spinnaker user is null.")
+      return accounts
```

Comment:
`return []` imo rather than `accounts` and expecting it's defaulted to `[]`.

then you could just do:
`return view.getServiceAccounts().collect { it.name }`